mkosi/config.py

```diff
@@ -2195,6 +2235,22 @@ def parse_config(path: Path, namespace: argparse.Namespace, defaults: argparse.N
         if not match_config(path, namespace, defaults):
             return False
 
+        if extras:
+            for s in SETTINGS:
```

Comment:
Yeah but I prefer using `s` as the variable name for the for loop everywhere, in which case this is necessary, so i'd prefer to keep this

Reply:
That's ok, we're not losing anything by not doing this, we're just not gaining anything (more type safety). Instead of the "Make mypy happy" comment please add a todo, though, add a todo to solve this with a type guard once we're on a sufficiently new Python version.